orchagent/port.h

```diff
@@ -172,6 +172,7 @@ class Port
     uint32_t            m_speed = 0;    // Mbps
     port_learn_mode_t   m_learn_mode = SAI_BRIDGE_PORT_FDB_LEARNING_MODE_HW;
     bool                m_autoneg = false;
+    bool                m_unreliable_los = false;
```

Comment:
@vdahiya12 can you treat the `unlos` as a serdes attribute ? i.e use port.serdes instead

Reply:
added part of serdes